libraries/BLE/src/ArduinoBLE.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Currently CurieBLE.h  supports the post 1.0.7 version - or CureiBLE with central
To support the deprecated API, we should also remove the central examples plus change some of the class names
so  void blePeripheralConnectHandler(BLEHelper& central) 
should probably revert to 
void blePeripheralConnectHandler(BLECentral & central)